src/lib/run-studies.ts

```diff
@@ -61,48 +91,7 @@ async function checkRunExists(client: ResourceGroupsTaggingAPIClient, runId: str
     return true
 }
 
-const ecsClient = new ECSClient()
-const taggingClient = new ResourceGroupsTaggingAPIClient()
-
-// Set in IaC
-const cluster = process.env.ECS_CLUSTER || ''
-const baseTaskDefinition = process.env.BASE_TASK_DEFINITION_FAMILY || ''
-const subnets = process.env.VPC_SUBNETS || ''
-const securityGroup = process.env.SECURITY_GROUP || ''
-
-// In case we want to test stuff without connecting to mgmt app
-const _managementAppSampleData = {
-    runs: [
-        {
-            runId: 'unique-run-id-3',
-            containerLocation: '084375557107.dkr.ecr.us-east-1.amazonaws.com/research-app:v1',
-            title: 'my-run-1',
-        },
-        {
-            runId: '1234',
-            containerLocation: '',
-            title: '',
-        },
-        {
-            runId: '456',
-            containerLocation: '',
-            title: '',
-        },
-    ],
-}
-
-// Wrap calls in a function to avoid layers of promise resolves
-const main = async (): Promise<void> => {
-    let ignoreAWSRuns = false
-    if (process.argv.includes('--help')) {
-        printHelp()
-        process.exit(0)
-    }
-    if (process.argv.includes('--ignore-aws')) {
-        console.log('Ignoring AWS existing runs')
-        ignoreAWSRuns = true
-    }
-
+export async function runStudies(ignoreAWSRuns: boolean): Promise<void> {
     // Uncomment to use local variables
     // const result = _managementAppSampleData
```

Comment:
Here the `_managementAppSampleData` is commented.

Reply:
I removed this with the change for the other comment, and while I was at it I noticed it might be nice to implement [this renaming](https://github.com/safeinsights/setup-app/pull/10/commits/dc9389a285f561581d43bdd91bcf8de7e6c33d3a) for clarity / readability.